src/psyclone/psyGen.py

```diff
@@ -5343,10 +5408,12 @@ def declare(self, name, datatype, shape=[], scope='local',
         :param str name: Name of the symbol.
         :param str datatype: Datatype of the symbol.
         :param list shape: Shape of the symbol in row-major order (leftmost \
-                       index is contiguous in memory). Each entry represents \
-                       an array dimension. If not None then it holds the \
-                       extent of that dimension. If it is an empy list it \
-                       represents an scalar.
+                           index is contiguous in memory). Each entry \
```

Comment:
Not a change in the PR, but I though column major was leftmost index being contiguous?

Reply:
Argh.. you are right, I copy-pasted this mistake several places, I fixed them now.